### PR TITLE
Populate metro and area code in IPv6 City lookups

### PR DESCRIPTION
**Pull request by horgh**

### CODE REVIEW

File: libGeoIP/GeoIP.c [hunk 1]
```diff
@@ -861,7 +861,7 @@ char * _GeoIP_iso_8859_1__utf8(const char * iso)
     char * p;
```
Comment (oschwald) [inline libGeoIP/GeoIP.c]: I wonder if we should drop this commit and just make a separate PR where we switch to `clang-format` with the style used in `geoipupdate`.

Comment (horgh) [reply libGeoIP/GeoIP.c]: Yeah, it was a bit huge. I've dropped it. There were a couple conflicts after doing so (the comments typo commit and the last one) but they were minor.
